jsre: TestNatto fails randomly

```
--- FAIL: TestNatto (0.02s)
    jsre_test.go:91: expected no error, got ReferenceError: 'msg' is not defined
    jsre_test.go:94: expected string value, got undefined
    jsre_test.go:99: expected 'testMsg', got 'undefined'
FAIL
FAIL    github.com/ethereum/go-ethereum/jsre    0.032s
```

--- a/jsre/jsre_test.go
+++ b/jsre/jsre_test.go
@@ -85,7 +85,7 @@ func TestNatto(t *testing.T) {
 	if err != nil {
 		t.Errorf("expected no error, got %v", err)
 	}
-	time.Sleep(time.Millisecond * 10)
+	time.Sleep(100 * time.Millisecond)
 	val, err := jsre.Run("msg")
 	if err != nil {
 		t.Errorf("expected no error, got %v", err)

